webapp/src/components/ProgramActivityDialog.tsx

```diff
@@ -1,85 +1,212 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
Since we're importing `useState()` and `useEffect()` here, let's do the same in `ProgramCalendar.tsx` and replace all the `React.(function name)` calls in that file with just the function names, so that things appear the same in both files.

Reply:
(We probably could also do the same with `Dispatch` and `SetStateAction`, too.)